src/test/java/com/softserveinc/ita/CheckoutPageTest.java

```diff
@@ -15,19 +15,22 @@
 
     @Test
     public void verifyCheckoutPageFunctionalityTest() {
-        homePage
-                .closeAdBanner()
-                .openLastProductOfSection("Акційні пропозиції");
+        var testSectionName = "Акційні пропозиції";
+
+        homePage.closeAdBanner();
+
+        var lastProductOfSection = homePage.getLastProductTitleOfSection(testSectionName);
+
+        homePage.openLastProductOfSection(testSectionName);
```

Comment:
here you can create var productPage, you will get it from openLastProductOfSection